### PR TITLE
Reverting distribution repository

### PR DESCRIPTION
**Pull request by Albertoimpl**
To re-enable snapshot publishing

### CODE REVIEW

File: ci/scripts/build-project.sh [hunk 1]
```diff
@@ -4,14 +4,18 @@ set -euo pipefail
 
 readonly ONLY_SHOW_STANDARD_STREAMS_ON_TEST_FAILURE="${ONLY_SHOW_STANDARD_STREAMS_ON_TEST_FAILURE:-true}"
 readonly SKIP_TESTS="${SKIP_TESTS:-false}"
-readonly DISTRIBUTION_REPOSITORY_OUTPUT="${DISTRIBUTION_REPOSITORY_OUTPUT:?must be set}"
 
-if [ "$SKIP_TESTS" == "true" ]; then
+# shellcheck source=common.sh
+source "$(dirname "$0")/common.sh"
+repository=$(pwd)/distribution-repository
+#if [ "$SKIP_TESTS" == "true" ]; then
```
Comment (spikymonkey) [inline ci/scripts/build-project.sh:11]: Is this intentional?

**Review by spikymonkey — CHANGES_REQUESTED** [review]

File: ci/scripts/build-project.sh (hunk 1 above)
Comment (Albertoimpl) [reply ci/scripts/build-project.sh]: ah, definitely not. I was testing it locally so wanted it to run faster. Just force-pushed it. Thanks!

**Review by spikymonkey — APPROVED** [review]
Looks great now, thanks!

**Comment by Albertoimpl** [comment]
Thank you @spikymonkey !

**Pull request merged**